src/renderer/component/recommendedContent/view.jsx

```diff
@@ -0,0 +1,76 @@
+// @flow
+import React from 'react';
+import FileTile from 'component/fileTile';
+import { FormRow, FormField } from 'component/common/form';
+import ToolTip from 'component/common/tooltip';
+import type { Claim } from 'types/claim';
+import { buildURI, parseURI } from 'lbry-redux';
+
+type Props = {
+  uri: string,
+  channelUri: ?string,
+  claimsInChannel: ?Array<Claim>,
+  autoplay: boolean,
+  setAutoplay: boolean => void,
+  fetchClaims: (string, number) => void,
+};
+
+export default class RecommendedContent extends React.PureComponent<Props> {
+  componentDidMount() {
+    const { channelUri, fetchClaims, claimsInChannel } = this.props;
+    if (channelUri && !claimsInChannel) {
+      fetchClaims(channelUri, 1);
+    }
+  }
+
+  render() {
+    const { claimsInChannel, autoplay, uri, setAutoplay } = this.props;
+
+    let recommendedContent;
+    if (claimsInChannel) {
+      recommendedContent = claimsInChannel.filter(claim => {
```

Comment:
This will be moved into a selector when I add recommended content from search results (my next PR)